apps/aecore/src/aec_conductor.erl

```diff
@@ -286,6 +304,10 @@ handle_call({get_block, Hash},_From, State) ->
     {reply, aec_conductor_chain:get_block(Hash, State), State};
 handle_call({get_block_by_height, Height},_From, State) ->
     {reply, aec_conductor_chain:get_block_by_height(Height, State), State};
+handle_call({get_block_range, GetFun},_From, State) when is_function(GetFun,
+                                                                     1)->
+    Resp = GetFun(State),
```

Comment:
Don't send the closure. Send an atom specifying which method to use. Applying general funs are unsafe.